tests/expected/cpp/33103-bug_858.cpp

```diff
@@ -0,0 +1,17 @@
+enum
+{
+   item1 = 123,
+   item2, // comment 2
+}
+
+enum
+{
+   item3,
+   item4,  // comment 4
+}
+enum { x, y,};
+enum { x, y = 0,};
+enum { x, y = 0,/*comment*/ };
```

Comment:
Thanks! :+1:

This is an... interesting result. However, I see that `sp_before_tr_emb_cmt` is still honored, so it is probably the best result.